cmd/virt-handler/virt_launcher.cil

```diff
@@ -0,0 +1,13 @@
+(block virt_launcher
```

Comment:
all of these policies, rules, or whatever they are called look reasonable to me.

I'm curious how you came up with them all. Did you have to look at some audit file to determine what needed to be permitted for the virt-launcher pod's processes?

basically, I'm trying to understand the process someone would need to go through to expand this logic in the future if we had to.

Reply:
Thanks for looking into this David!

Yes, basically I went over each of our tests looking for audit denials and general failure trying to identify whether the failure was caused by SELinux.  Then I've enabled it in our policy.
After having all of the rules in place I've minimized it to the current policy.
It comes down to the basic container_t label with enabled networking - similar policy exists for virtual machines svirt_*
In addition, I have to allow access to the generated HostDisk and EmptyDisk- that's why you see allowed `container_share_t` and `container_var_run_t`
`tmp_t` is needed for DataVolume
`kernel_t` is needed for libvirt to check is certain modules are loaded
`mtrr_device_t` is also required by libvirt